DEVELOPMENT.md

```diff
@@ -0,0 +1,200 @@
+# Apisonator
+
+## Development environment set up
+
+### Prerequisites
+
+* Docker (requires version 1.10.0 or later)
+
+To learn how to run the project once the environment has been set up, refer to
+the "Running" section.
+
+The first thing you will need is cloning the project:
+> `$ git clone git@github.com:3scale/apisonator.git`
+
+Next cd into the directory, `cd apisonator`.
+
+### Containerized environment
+
+#### With Docker
+
+This requires GNU Make and has a single step:
+
+1. Run: `make dev`
+
+This command will take care of downloading and building all dependencies. Once
+that is done, the process will be way faster the next time.
+
+The project's source code will be available in `~/apisonator` and sync'ed with
+your local apisonator directory, so you can edit files in your preferred
+environment and still be able to run whatever you need inside the Docker
+container.
+
+The listener service port (3000 by default) is automatically forwarded to the host machine. The `make dev` command can be executed with the environment variable `PORT` set to
```

Comment:
really long lines here, hard to edit - do you have a way to autowrap long lines so that a newline char is inserted when too long?

Reply:
I have updated the README and DEVELOPMENT documents to try to stick the lines more or less at 80 characters. In my IDE I have a vertical line as a guide to this